dts/bindings/display/panel-controller.yaml

```diff
@@ -0,0 +1,117 @@
+# Copyright 2023 NXP
+# Common fields for display controllers driving a panel
+
+include: display-controller.yaml
```

Comment:
we should keep Linux dt bindings hierarchi, ie, `display/panel/panel-timing.yaml` here

Reply:
Binding has been moved